features/view_specs/view_spec.feature

```diff
@@ -212,3 +212,24 @@ Feature: view spec
       """
     When I run `rspec spec/views`
     Then the examples should all pass
+
+  Scenario: path helpers should take precedence over asset helper
```

Comment:
> It would be better to write this as a unit spec as it's not applicable to our documentation really

I'd love some help to get this rewritten as unit test as well. I tried to do something like this in the spec and it doesn't work ...

```ruby
# in spec/rspec/rails/example/view_example_group_spec.rb
    describe "routes helpers collides with aset helpers" do
      it "prefers routes helper" do
        group = RSpec::Core::ExampleGroup.describe 'show.html.erb' do
          include ViewExampleGroup

          it "passes" do
            ### this never gets called
            puts "TESTING"
          end
        end

        group.run NullObject.new
      end
    end
```

Am I doing this right?